apps/src/blockly/addons/cdoFieldParameter.ts

```diff
@@ -239,31 +241,31 @@ export default class CdoFieldParameter extends GoogleBlockly.FieldVariable {
   };
 
   /**
-   * Prompt the user for a variable name or delete a variable.
-   * @param promptText description text for window prompt
-   * @param confirmButtonLabel Label of confirm button, e.g. "Rename"
-   * @param callback with parameter (text) of new name
-   * @param prompt Whether to prompt for a string value
-   * @param isDangerCancel Should cancel button has a danger type
-   * @param defaultText default input text for window prompt
+   * Prompt the user to name or delete a parameter.
+   * @param {object} options The options object.
+   * @param {string} options.promptText Description text for window prompt.
+   * @param {string} options.confirmButtonLabel Label of confirm button, e.g., "Rename".
+   * @param {function} options.callback Callback with parameter (text) of new name.
+   * @param {boolean} options.prompt Whether to prompt for a string value.
+   * @param {string} [options.defaultText] Default input text for window prompt.
    */
-  static modalDialogName = function (
-    promptText: string,
-    confirmButtonLabel: string,
-    callback: (newName: string) => void,
-    prompt: boolean,
-    isDangerCancel: boolean,
-    defaultText?: string
-  ) {
+  static parameterPrompt = function (options: {
```

Comment:
mind making a type for the options object? Up to you if you want to keep the javadoc above since the parameters would be documented in a type.